pkgs/by-name/ca/carl/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "carl";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "b1rger";
+    repo = "carl";
+    rev = "v${version}";
+    hash = "sha256-+l11eP+1qKrWbZhyUJgQ8FgQ+2rncx778F5RPzCfvV4=";
+  };
+
+  doCheck = false;
```

Comment:
Please enable tests

Reply:
I've explained in the description that tests are not functional, and I reported it in an upstream issue.

Reply:
Seems like some kind of nix issue. I cloned and ran the test on latest master and v0.3.1 and all ran successfully.

Reply:
I think I found the culprit, if it works I may fix the tests for a handful of other packages too.

https://github.com/NixOS/nixpkgs/pull/368593